backend/api/messages.py

```
@@ -0,0 +1,468 @@
+        {
+            "id": user.id,
+            "full_name": user.full_name,
+            "email": user.email,
+            "role": user.role
+        }
+        for user in users
+    ]
+
+@router.get("/cohorts/", response_model=List[dict])
+async def get_cohorts(
+    current_user: User = Depends(get_current_user),
```

Ruff (0.13.3)

Unused function argument: `current_user`

(ARG001)